bundle/config/mutator/translate_paths.go

```diff
@@ -180,9 +186,9 @@ func (t *translateContext) rewritePath(
 }
 
 func (t *translateContext) translateNotebookPath(ctx context.Context, literal, localFullPath, localRelPath string) (string, error) {
-	nb, _, err := notebook.DetectWithFS(t.b.SyncRoot, filepath.ToSlash(localRelPath))
+	nb, _, err := notebook.DetectWithFS(t.b.SyncRoot, localRelPath)
 	if errors.Is(err, fs.ErrNotExist) {
-		if filepath.Ext(localFullPath) != notebook.ExtensionNone {
+		if path.Ext(localFullPath) != notebook.ExtensionNone {
```

Comment:
do we need to perform similar clean up across the whole codebase, or at least in bundle?

```
(main) ~/work/cli % git grep filepath bundle | wc -l
     507
```

Reply:
There are a few other places where we do the same (e.g. sync includes), but definitely not everywhere.

The `filepath` package must be used for relative path computations because on Windows it makes sure it doesn't escape the drive or UNC path. I recall that also for globbing the input paths need to use the platform-native separator or they don't work.